### PR TITLE
DRAFT Compiler FE: Enable SelectorV2 Op

### PR DESCRIPTION
**Pull request by ai-moiseev**
Enable SelectV2 operator.

For #2951

### CODE REVIEW

File: compiler/luci/import/src/GraphBuilderRegistry.cpp [hunk 1]
```diff
@@ -107,6 +107,7 @@ GraphBuilderRegistry::GraphBuilderRegistry()
   CIRCLE_NODE(SCATTER_ND, CircleScatterNdGraphBuilder);                                    // 122
   CIRCLE_NODE(SEGMENT_SUM, CircleSegmentSumGraphBuilder);                                  // 125
   CIRCLE_NODE(SELECT, CircleSelectGraphBuilder);                                           // 64
+  CIRCLE_NODE(SELECT_V2, CircleSelectV2GraphBuilder);                                      // 123
```
Comment (seanshpark) [inline compiler/luci/import/src/GraphBuilderRegistry.cpp:110]: please remove below `// BuiltinOperator_SELECT_V2 = 123,` line

**Comment by seanshpark** [comment]
Looks  Good!, just one minor comment :)

File: compiler/luci/import/src/GraphBuilderRegistry.cpp (hunk 1 above)
Comment (ai-moiseev) [reply compiler/luci/import/src/GraphBuilderRegistry.cpp:110]: Done :)

**Comment by binarman** [comment]
Everything is merged, thank you!

**Pull request closed without merging**